xml/System.Diagnostics/DiagnosticSource.xml

```diff
@@ -128,10 +128,13 @@
         <Parameter Name="args" Type="System.Object" Index="1" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="activity">To be added.</param>
-        <param name="args">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="activity">The <see cref="T:System.Diagnostics.Activity" /> to be started.</param>
+        <param name="args">An object that represent the value being passed as a payload for the event.</param>
+        <summary>Starts an <see cref="T:System.Diagnostics.Activity" /> and writes start event.
+The <see cref="T:System.Diagnostics.Activity" /> describes the logical operation, its context and parent relation; the current <see cref="T:System.Diagnostics.Activity" /> flows through the operation processing.
+This method starts the given <see cref="T:System.Diagnostics.Activity" /> (maintains global <see cref="P:System.Diagnostics.Activity.Current" /> <see cref="T:System.Diagnostics.Activity" /> and <see cref="P:System.Diagnostics.Activity.Parent" /> for the given <see cref="T:System.Diagnostics.Activity" />) and notifies consumers that new <see cref="T:System.Diagnostics.Activity" /> was started. Consumers could access <see cref="P:System.Diagnostics.Activity.Current" /> to add context and/or augment telemetry.
+Producers may pass additional details to the consumer in the payload.</summary>
+        <returns>The started <see cref="T:System.Diagnostics.Activity" /> for convenient chaining.</returns>
         <remarks>To be added.</remarks>
```

Comment:
```suggestion
        <remarks>  
          <format type="text/markdown"><![CDATA[  

The <see cref="T:System.Diagnostics.Activity" /> describes the logical operation, its context, and parent relation; the current <see cref="T:System.Diagnostics.Activity" /> flows through the operation processing. 

This method starts the given <xref:System.Diagnostics.Activity>, maintains a global <xref:System.Diagnostics.Activity.Current?displayProperty=nameWithType> and <xref:System.Diagnostics.Activity.Parent?displayProperty=nameWithType> activity for the given <xref:System.Diagnostics.Activity>, and notifies consumers that a new <xref:System.Diagnostics.Activity> was started. Consumers can access <xref:System.Diagnostics.Activity.Current?displayProperty=nameWithType> to add context and/or augment telemetry.

Producers may pass additional details to the consumer in the payload.

                  ]]></format>
         </remarks>
```

I assumed that the second paragraph just means to say that the method maintains a global current and parent activity for this activity?

Reply:
That's how it sounds to me too. @wtgodbe can confirm.

Reply:
@rpetrusha I tweaked your suggestion a bit because there was one extra `<see cref` that I converted to `<xref` and removed the type/member id:

```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[
           
The <xref:System.Diagnostics.Activity> describes the logical operation, its context, and parent relation; the current <xref:System.Diagnostics.Activity> flows through the operation processing.

This method starts the given <xref:System.Diagnostics.Activity>, maintains a global <xref:System.Diagnostics.Activity.Current?displayProperty=nameWithType> and <xref:System.Diagnostics.Activity.Parent?displayProperty=nameWithType> activity for the given <xref:System.Diagnostics.Activity>, and notifies consumers that a new <xref:System.Diagnostics.Activity> was started. Consumers can access <xref:System.Diagnostics.Activity.Current?displayProperty=nameWithType> to add context and/or augment telemetry.

Producers may pass additional details to the consumer in the payload.

           ]]></format>
         </remarks>
```